atomics/include/desul/atomics/Compare_Exchange_OpenMP.hpp

```diff
@@ -7,6 +7,10 @@ SPDX-License-Identifier: (BSD-3-Clause)
 */
 #ifndef DESUL_ATOMICS_COMPARE_EXCHANGE_OPENMP_HPP_
 #define DESUL_ATOMICS_COMPARE_EXCHANGE_OPENMP_HPP_
+#include <omp.h>
+
+#include <cstdio>
```

Comment:
I don't see why we should include that header.

Reply:
removed it